exer2-http-airport-json-parse.py

```diff
@@ -5,21 +5,20 @@
 response = requests.get(url)
 
 json_response = response.json()
-data = json_response["meta"]["view"]["columns"]
+raw_data = json_response["meta"]["view"]["columns"]
 # display the average / max number of passengers per month per airline traveling through BWI airport
 
-print(
-    "========== ============================== ==================== =========" "\n"
-    "ID         Airline                        Average              Largest " "\n" 
-    "---------- ------------------------------ -------------------- ---------")
-for i in data:
+sanitized_data = list()
+
+# Collect Data
```

Comment:
I like to separate the sanitizing of the data set from the display.  Here we will only create a new data structure which can be displayed later. Not sure if there's a right/wrong, but just another option.